packages/cli/src/program.ts

```diff
@@ -0,0 +1,319 @@
+import { Command } from 'commander';
```

Comment:
This file is duplicated in `packages/cli/lib/program.ts`. In this case, I prefer this location.